src/Symfony/Component/HttpKernel/EventListener/ExceptionListener.php

```diff
@@ -30,6 +31,7 @@ class ExceptionListener implements EventSubscriberInterface
 {
     protected $controller;
     protected $logger;
+    public static $httpStatusCodeLogLevel = [];
```

Comment:
constructor arg; to be configured by the framework?

Reply:
same here: this should not be a public static, but a regular private property IMHO

Reply:
yep for sure - see my comment above
> This is just demonstrating the direct approach, open to feedback and suggestions on how something like this should be better configured in yaml

Reply:
Fixed. But I'll have to follow up by integrating into DI